modules/apps/headless/headless-delivery/headless-delivery-api/src/main/java/com/liferay/headless/delivery/dto/v1_0/util/ContentFieldUtil.java

```diff
@@ -115,16 +115,16 @@ public static ContentField toContentField(
 
 						Map<String, ContentFieldValue> map = new HashMap<>();
 
-						Map<Locale, String> valueValues = Optional.ofNullable(
-							ddmFormFieldValue.getValue()
-						).map(
-							Value::getValues
-						).orElse(
-							Collections.emptyMap()
-						);
+						Value value = ddmFormFieldValue.getValue();
+
+						Map<Locale, String> values = value.getValues();
+
+						if (values == null) {
+							values = Collections.emptyMap();
```

Comment:
rename valueValues to value, also assign `Collections.emtpyMap()` to values if it is null